xml/xml-quiz.md

```diff
@@ -206,7 +206,7 @@
 
 #### Q20. You want to convert a large XML file into CSV format. You did not create the XML file, so you are not familiar with all of the syntax. What will help you get the best insight into the file contents?
 
-- [ ] XSLT
+- [x] XSLT
```

Comment:
XSLT is a transformation process. But the question is not about the transformation, it's about understanding the XML. XSD is the correct one here - that's the schema document, that describes the XML.

Reply:
Thanks for review @ruslanbes I'll update the answer and put your explanation in *md file